src/extensions/renderer/canvas/drawing-images.js

```diff
@@ -81,14 +81,14 @@ CRp.drawInscribedImage = function( context, img, node, index ){
 
   var x = (nodeX - nodeW / 2); // left
   if( getIndexedStyle( node, 'background-position-x', 'units', index ) === '%' ){
-    x += (nodeW - w) * getIndexedStyle( node, 'background-position-x', 'pfValue', index );
+    x += (node.outerWidth() - w) * getIndexedStyle( node, 'background-position-x', 'pfValue', index );
```

Comment:
It should replace `nodeW` instead I think, as the same value needs to be used for clipping, repeating, etc.